_posts/2019-06-25-lambda-quickstart.md

```diff
@@ -0,0 +1,320 @@
+---
+title: How to configure OpenFaaS functions to execute on AWS Lambda
+description: In a follow up article Edward Wilde shows how to configure OpenFaaS functions to execute on AWS Lamda using the new faas-lambda provider.
+date: 2019-06-25
+image: /images/faas-quickstart/backdrop.jpg
+categories:
+  - lambda
+  - automation
+  - tutorial
+author_staff_member: ed
+dark_background: true
+---
+
+In this post we are going to follow the steps necessary to configure OpenFaaS to execute function on AWS Lambda using 
+the new faas-lambda provider. See the previous blog post [announcing support for AWS Lambda] for an overview of the 
+technology.
+
```

Comment:
I think the introduction could be better. Please look at the intro from the other blog posts. Remember the first line appears on the roll-up, so it should be short, followed by around 200 words summarising the post. See the changes I made in the edit of the announcing blog post.